docs/machine-learning/how-to-guides/how-to-use-the-automl-api.md

```diff
@@ -12,6 +12,15 @@ Automated machine learning (AutoML) automates the process of applying machine le
 > [!NOTE]
 > This topic refers to the automated machine learning API for ML.NET, which is currently in preview. Material may be subject to change.
 
+## AutoML Implementation
+
+AutoML has collaborated with Microsoft Research teams [Neural Network Intelligence](https://github.com/Microsoft/nni) (NNI) and [Fast and Lightweight AutoML](https://github.com/Microsoft/flaml) (FLAML). These teams help provide additional benefits to the AutoML implementation. These benefits include:
```

Comment:
@jwood803 looking at this section, while technically it is true that AutoML in tooling like Model Builder leverages NNI / FLAML, I don't think those implementations are part of the AutoML API. @LittleLittleCloud can you confirm?

Reply:
Nope not part of AutoML API yet